timeless/__init__.py

```diff
@@ -13,7 +13,7 @@
 from timeless.cache import CACHE
 from timeless.mail import MAIL
 from timeless.db import DB
-from timeless.sync.celery import make_celery
```

Comment:
@vladarefiev Why did you move this file? If it's only used for syncing, maybe it makes more sense to stay under ``timeless.sync``. or?

Reply:
@amihaiemil Actually it is not. `celery` is framework for async jobs.  particularly `celery.py` should consist of settings for jobs/queues and may be schedular for them. So, seems more logic to put in root. But you are right, every modular may contain file with jobs/tasks, usually it called `tasks.py`, we have already had this in poster module.